content/zh/docs/concepts/workloads/pods/init-containers.md

```diff
@@ -74,7 +74,7 @@ If you specify multiple init containers for a Pod, Kubelet runs each init contai
 
 Init 容器支持应用容器的全部字段和特性，包括资源限制、数据卷和安全设置。 然而，Init 容器对资源请求和限制的处理稍有不同，在下面 [资源](#资源) 处有说明。
 
-同时 Init 容器不支持 Readiness Probe，因为它们必须在 Pod 就绪之前运行完成。
+同时 Init 容器不支持 Lifecycle、LivenessProbe、ReadinessProbe和StartupProbe，因为它们必须在 Pod 就绪之前运行完成。
```

Comment:
pls also sync English content on line 69.

Reply:
ok. thanks, I am done it .